lua/roslyn/sln/api.lua

```diff
@@ -0,0 +1,44 @@
+local M = {}
+
+local sysname = vim.uv.os_uname().sysname:lower()
+local iswin = not not (sysname:find("windows") or sysname:find("mingw"))
+
+---@param solution string Path to solution
+---@return string[] Table of projects in given solution
+function M.projects(solution)
```

Comment:
I can see this being useful for potential use of third party plugins or in my own config or something, so therefore I am exposing this now already.

I want to experiment with this myself in my own config